apps/web/src/features/spaces/hooks/useIsLastActiveAdmin.ts

```diff
@@ -0,0 +1,19 @@
+import { useMemo } from 'react'
+import { isActiveAdmin, isAdmin, useSpaceMembersByStatus } from './useSpaceMembers'
+import type { Member } from '@safe-global/store/gateway/AUTO_GENERATED/spaces'
+import { useCurrentMembership } from './useSpaceMembers'
+
+export const useAdminCount = (members?: Member[]) => {
+  const { activeMembers } = useSpaceMembersByStatus()
+  const membersToUse = members ?? activeMembers
+  return useMemo(() => membersToUse.filter(isAdmin).length, [membersToUse])
+}
+
+export const useIsLastActiveAdmin = (member?: Member) => {
```

Comment:
It looks like this is only used in one place without passing a member prop. Can we remove it for now and always use `useCurrentMembership`?